Add support for retrieving the VM user name and password from KeyVaul…

scotmoor opened the pull request:
…t when creating Scale Sets

Commit:
Add support for retrieving the VM user name and password from KeyVault when creating Scale Sets

JargoonPard commented on clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/common/model/KeyVaultSecret.groovy:
```diff
@@ -0,0 +1,29 @@
+
+  KeyVaultSecret(String secretName, String subscriptionId, String resourceGroupName, String vaultName) {
+    keyVault = new IdRef("/subscriptions/$subscriptionId/resourceGroups/$resourceGroupName/providers/Microsoft.KeyVault/vaults/$vaultName")
+    this.secretName = secretName
```
inconsistent style with previous line that doesn't use "this"

JargoonPard commented on clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy:
```diff
@@ -297,8 +307,13 @@ class AzureResourceManagerClient extends AzureBaseClient {
+  }
+
+  @Canonical
+  static class ReferenceParameter {
```
ValueParameter class is private. Should this one be private as well? They should be consistent.

JargoonPard commented:
LGTM

Commit:
Added stackdriver

cfieber committed:
Revert "Upgrade to Spring Boot 1.4.1.RELEASE"

tomaslin committed:
titus - terminate and shrink operation support

Commit:
(aws) Cache instance health for both vpc and non-vpc load balancers. (#1163)

Commit:
(google) Cache and propagate autoscaling messages. (#1165)

jtk54 committed:
provider/google: Implemented SSL LB caching agent. (#1162)

tomaslin committed:
Merge pull request #1164 from tomaslin/titus-terminate-and-shrink

titus- detachInstances operation support

jtk54 committed:
provider/google: Prevent name/key collisions in health check agent. (#1167)

Also provide context which health check 'family' a health check
belongs to.

jtk54 committed:
provider/google: Prevent NPE in backend service caching agent. (#1168)

tomaslin committed:
titus - supress error messages from legacy server groups

tomaslin committed:
Merge pull request #1169 from tomaslin/check-for-null-application-names-titus-cache

titus - supress error messages from legacy server groups

Commit:
Merge pull request #1161 from ewiseblatt/stackdriver

Added stackdriver

jtk54 committed:
provider/google: Added SSL LB disabled state calculations. (#1171)

Cleans up the cases of this calculation as well.

jtk54 committed:
provider/google: Fix ILB protocol in Upsert. (#1172)

jtk54 committed:
provider/google: Implemented SSL LB deploy and clone. (#1173)

lwander committed:
Errorconfig handles XSS vulnerabilites by removing requested paths (#1174)

lwander committed:
(kubernetes) fix duplicate IPSs in RS spec (#1176)

jtk54 committed:
provider/google: Support custom networks/subnets in ILB upsert. (#1175)

jtk54 committed:
provider/cf: Ignore test with live calls until TravisCI issue corrected. (#1178)

ajordens committed:
Optimize Titus<->AWS security group lookup

tomaslin committed:
Merge pull request #1182 from ajordens/master

Optimize Titus<->AWS security group lookup

lwander committed:
(docker) reload passwordfile everytime credentials are loaded (#1180)

jtk54 committed:
provider/google: Update instance metadata on L7 upsert. (#1177)

* provider/google: Update instance metadata on L7 upsert.

* provider/google: Fix instance metadata for backend services removed in L7 upsert.

jtk54 committed:
provider/google: Separate Redis keyspaces for backendServices and regionBackendServices. (#1179)

Also added a 'kind' attribute to GoogleBackendService to distinguish
between the API families.

lwander committed:
(kubernetes) ~20x faster application load (#1183)
